### PR TITLE
Sw change put `/events/{id}/artists` to accept an `artists` param which will be the array

### PR DESCRIPTION
**Pull request by SWvheerden**
Fixes issue #403

### CODE REVIEW

File: api/src/controllers/events.rs [hunk 1]
```diff
@@ -85,12 +85,17 @@ pub struct AddArtistRequest {
...
-#[derive(Deserialize, Debug)]
+#[derive(Deserialize, Debug, Default)]
 pub struct UpdateArtistsRequest {
     pub artist_id: Uuid,
     pub set_time: Option<NaiveDateTime>,
 }
 
+#[derive(Deserialize, Debug, Default)]
+pub struct UpdateArtistsRequestList {
+    pub update_req: Vec<UpdateArtistsRequest>,
```
Comment (Timo614) [inline api/src/controllers/events.rs]: Shouldn't we make this `artists`?

Comment (SWvheerden) [reply api/src/controllers/events.rs]: Its not updating the complete artist. 
Just this: pub set_time: Option<NaiveDateTime>
I beleive doing it this way is a much smaller request. 

This was done on request for @Jasonvdb ,  So perhaps he and @mikethetike  can make a comment here?

Comment (Timo614) [reply api/src/controllers/events.rs]: I was just commenting on the name of the parameter not its type as the associated issue and this PR's title call it `artists` but it's named `update_req` here.

Comment (SWvheerden) [reply api/src/controllers/events.rs]: Well I am open to change the name, but I would steer away from "artist" as that would conflict with the artist struct. 

as to the PR name, its the exact name of the issue that was open

Comment (mikethetike) [reply api/src/controllers/events.rs]: I agree with @Timo614 here. I think `artists` will make more sense to the front end developers consuming this service

**Pull request merged**